### ISSUE TITLE
Displaying ELO with LLR statistics

### ISSUE THREAD

**Issue by Chess13234** [glinscott/fishtest#514]
I find it strange that ELO is only displayed on the live ELO page, while LOS is available.
Could it be added within the stat box instead of -2.94,2.94(which is the same for every single test) ?
Example:
```
LLR: -0.75 ( +1.22 ELO) {-1.00,3.00}
Total: 19537 W: 3718 L: 3683 D: 12136
Ptnml(0-2): 263, 2205, 4818, 2181, 292
```

**Comment by MJZ1977** [comment]
This proposal can be useful because it avoids clicking each time to live page.

**Comment by vondele** [comment]
but would only make sense if the Elo interval is displayed. Just looking at the Elo without the error bounds is misleading.

**Comment by Chess13234** [comment]
@vondele what about this form?
LLR: -21.4% (-3.13 | +0.22 ELO |2.31) {-1.00,3.00}
Total: 19537 W: 3718 L: 3683 D: 12136
Ptnml(0-2): 263, 2205, 4818, 2181, 292

**Comment by vondele** [comment]
yes, some variant is possible, but LLR in % is strange, it is a log-likelihood ratio so just a number IMO. I agree with removing our -2.94, 2.94 bounds. I might have a look to see what is possible later this week.

**Comment by Chess13234** [comment]
@vondele its a "percent to completion".
-100% is failed test, +100% is passing.
when you see 1.11 without(-2.94 2.94 ) you have no idea if its close to completion or not.
Also if  (-2.94 2.94 ) change in the future, the percentage display will still remain accurate because it displays the range(-100% 100%) for any bounds.
